style.css

```diff
@@ -0,0 +1,161 @@
+.header-style {
+  height: 150px;
+}
+
+.logo {
+  position: static;
+  width: 100px;
+  height: 20px;
+  left: 12px;
+  top: 50px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  text-decoration: none;
+
+  /* identical to box height, or 111% */
+  display: flex;
+  align-items: center;
+  text-align: center;
+
+  /* Primary/B400 */
+  color: #6070ff;
+}
+
+.nav-image {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  top: calc(5% - 80px / 2);
+  border-radius: 8px;
+}
+
+.intro {
+  /* Primary Text */
+  position: static;
+  width: 327px;
+  height: 100px;
+  left: 24px;
+  top: 80px;
+
+  /* Header / 2 */
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+
+  /* or 130% */
+  display: flex;
+  align-items: center;
+
+  /* Neutrals/N800 */
+  color: #172b4d;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.professional-details {
+  /* Primary Text */
+  position: static;
+  width: 327px;
+  height: 146px;
+  left: 24px;
+  top: 300px;
```

Comment:
- [ ] Kindly follow the [Figma-design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)  as much as you can
- [ ] I would suggest you make the menu size as small as the Figma design.
- [ ] Make sure that your social media icons are inline. Your LinkedIn icon is on a different line. kindly fix it.
- Your design
![your-design](https://user-images.githubusercontent.com/81727988/164289446-f93e9c8c-e2ff-4640-97c3-891600ec7a8e.png)
- Figma design
![figma-design](https://user-images.githubusercontent.com/81727988/164289504-24ea89f9-34f1-43aa-9874-4c4b3cc1e449.png)